.github/workflows/build-and-test-arm.yml

```diff
@@ -50,7 +49,7 @@ jobs:
         run: make -j4 gotest
   arm-unittest:
     if: ${{ github.actor != 'dependabot[bot]' && (contains(github.event.pull_request.labels.*.name, 'Run ARM') || github.event_name == 'push' || github.event_name == 'merge_group') }}
-    runs-on: actuated-arm64-4cpu-4gb
```

Comment:
what was this one? are we skipping this in favor of macos-14, or are they equivalent?

Reply:
the [actuated runners](https://github.com/self-actuated/actuated) were used to test arm only. the macos-14 provides the same coverage AFAIK